packages/@okta/vuepress-site/docs/reference/api/event-hooks/index.md

```diff
@@ -16,6 +16,9 @@ For general information on event hooks and how to create and use them, see [Even
 
 Explore the Event Hooks API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/b02d234a2af183981254)
 
+> **Important:** In the future the Events API will not be tracking new event types added to the System Log API. For this reason we highly recommend migrating to the System Log API.
+<br>
+
```

Comment:
This page is about Event Hooks, not the Events API.

Reply:
If I'm understanding correctly, I think you're saying that it would be helpful to mention that the event objects that are delivered by Event Hooks are the new kind of event objects used in System Log API, not the old kind of event objects used in the Events API.

That would probably belong on the Overview page for Event Hooks (https://developer.okta.com/docs/concepts/event-hooks/), rather than this page, which is just reference documentation for the management API.

Reply:
It is unclear if we can continue to implement okta event hooks to listen for user events. It seems that we should use the System Log API as any new events or event types will only be added to the System Log.

Reply:
kcbrewron, just to confirm: you can certainly go ahead and implement Okta Event Hooks to listen for events. The events that are delivered by Event Hook are not being phased out or deprecated. The events delivered are Okta System Log API events, while the ones that are being phased out are Okta Event Log API events.

If there is confusion, I can add a clarification to the Event Hooks overview page saying that the events that are delivered are Okta System Log API events, not Okta Event Log API events.

Let me know if this makes sense.